src/components/accordion/accordion.component.js

```diff
@@ -205,10 +214,8 @@ Accordion.propTypes = {
   subTitle: PropTypes.string,
   /** Sets accordion size */
   size: PropTypes.oneOf(['large', 'small']),
-  /** Adds additional top and bottom padding */
-  customPadding: PropTypes.number,
```

Comment:
This needs to be a breaking change now that this prop is being removed

Reply:
You are right, I will add a message while squashing